tool/src/test/java/org/wildfly/security/tool/FileSystemEncryptRealmCommandTest.java

```diff
@@ -114,6 +114,12 @@ public void testBulk() throws Exception {
         }
     }
 
+    @Test
+    public void testBulkWithoutNames() throws Exception {
+        String descriptorFileLocation = "./target/test-classes/bulk-encryption-conversion-desc-without-names";
+        runCommand(descriptorFileLocation, 0);
```

Comment:
@lvydra Looks good! Just a question, the resulting CLI file should be deleted after the test. Can we add a afterclass method to do it?